.github/workflows/slither.yml

```diff
@@ -22,16 +22,27 @@ jobs:
         with:
           submodules: recursive
 
-      - name: Run Slither
-        uses: crytic/slither-action@6ef3a33e56de4e8f59488cf60858b5c1bf4967c0
-        id: slither
-        continue-on-error: true
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+
+      - name: Install Slither
+        run: |
+          pip install slither-analyzer
+
+      - name: Install Foundry
+        uses: foundry-rs/foundry-toolchain@v1
         with:
-          target: "contracts/"
-          sarif: results.sarif
-          slither-config: "contracts/slither.config.json"
+          version: nightly
+          cache: false
+
+      - name: Run Slither
+        run: |
+          cd contracts
+          slither ./ --fail-none --sarif ./results.sarif
 
       - name: Upload SARIF file
         uses: github/codeql-action/upload-sarif@v2
         with:
-          sarif_file: ${{ steps.slither.outputs.sarif }}
+          sarif_file: contracts/results.sarif
```

Comment:
I'm assuming that the above step creates a file `contracts/results.sarif`. We should add this to the `.gitignore` if it's not there already.

Reply:
the `results.sarif` file should be within the github action runner

Reply:
Technically we should be able to run slither locally if we wanted to. In which case we would want it ignored.